www-client/luakit/luakit-9999.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="A fast, light, simple to use micro-browser using WebKit and Lua."
+HOMEPAGE="https://aidanholm.github.io/luakit/"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="git://github.com/aidanholm/luakit.git"
```

Comment:
aidanholm has transitioned two the luakit account as the new upstream (merged), iirc